Simple comment breaks PHPUnit ExceptionAnnotationRector

# Bug Report

<!-- First, thank you for reporting a bug. That takes time and we appreciate that! -->

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version |  Rector v0.7.16            |
| Installed as   | composer global       - Windows 10, php 74              |

<!-- Please describe your problem here. -->

Simple comment line between two methods, like // test break this rector 

`>  rector process  ./example/`

```

 [ERROR] Could not process "example/ExceptionMessageTest.php" file by "Rector\PHPUnit\Rector\ExceptionAnnotationRector",
         due to:                                                                                                        
         "Call to a member function hasByName() on null".  

```

## Minimal PHP Code Causing Issue

https://getrector.org/demo/5f9a2a8a-34fd-471d-9869-41b52cb1e888#result

and attached as zip. 
[example.zip](https://github.com/rectorphp/rector/files/4491167/example.zip)

please open again

thanks for reminding me about instalation, but problem is not connected. 

```
vendor/bin/rector process ./example/
Rector v0.7.16
Config file: rector.yaml

 3/3 [============================] 100%


 [ERROR] Could not process "example/ExceptionMessageTest.php" file by "Rector\PHPUnit\Rector\ExceptionAnnotationRector",
         due to:
         "Call to a member function hasByName() on null".
```

```
php -v
PHP 7.4.4 (cli) (built: Mar 17 2020 13:49:19) ( ZTS Visual C++ 2017 x64 )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Xdebug v2.9.4, Copyright (c) 2002-2020, by Derick Rethans
```

[example.zip](https://github.com/rectorphp/rector/files/4501514/example.zip)

Problem is active even on master: 

```
 ./vendor/bin/rector process ./example/
Rector dev-master@7e2394e
Config file: rector.yaml


 3/3 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


 [ERROR] Could not process "example/ExceptionMessageTest.php" file by "Rector\PHPUnit\Rector\ExceptionAnnotationRector",
         due to:
         "Call to a member function hasByName() on null".
```


[example-master.zip](https://github.com/rectorphp/rector/files/4501653/example-master.zip)

--- a/rules/phpunit/src/Rector/ExceptionAnnotationRector.php
+++ b/rules/phpunit/src/Rector/ExceptionAnnotationRector.php
@@ -83,8 +83,11 @@ PHP
             return null;
         }
 
-        /** @var PhpDocInfo $phpDocInfo */
+        /** @var PhpDocInfo|null $phpDocInfo */
         $phpDocInfo = $node->getAttribute(AttributeKey::PHP_DOC_INFO);
+        if ($phpDocInfo === null) {
+            return null;
+        }
 
         foreach (self::ANNOTATION_TO_METHOD as $annotation => $method) {
             if (! $phpDocInfo->hasByName($annotation)) {

--- a/rules/phpunit/tests/Rector/ExceptionAnnotationRector/Fixture/skip_method_with_null_php_doc.php.inc
+++ b/rules/phpunit/tests/Rector/ExceptionAnnotationRector/Fixture/skip_method_with_null_php_doc.php.inc
@@ -1,0 +1,21 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\ExceptionAnnotationRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+class SkipMethodWithNullPhpDoc extends TestCase
+{
+    public function testLiteralMessage(): void
+    {
+        $this->expectException('Exception');
+        throw new \Exception('A literal exception message');
+    }
+
+    // test
+
+    public function testPartialMessageBegin(): void
+    {
+        throw new \Exception('A partial exception message');
+    }
+}

